### PR TITLE
Allow the use of .graphql or .graphql.js

### LINKED ISSUE #2152: Rename custom schema.graphql to schema.graphql.js
**What is the expected behavior?**
I'd like the custom graphql schema defined in ```./api/*/config/schema.graphql``` to allow the use of a ```.js``` file ending. 

Since these files are not actually .graphql formatted files it causes a lot of issues with syntax highlighting in IDE's. Also they aren't truly .graphql files so they should really be labelled as such.

I'd propose that to acheive backward compatibility that the system would allow for either ```schema.graphql``` or ```schema.graphql.js``` as a valid file name.

### PR DESCRIPTION
**Pull request by greatwitenorth**
Previously schema files could only end in .graphql files. This caused a problem with IDE's since these files are actually javascript files. This pull request should also detect ```.graphql.js``` files while keeping backward compatibility with the ```.graphql``` extension.

My PR is a:

💅 Enhancement 

Main update on the:

Framework 

Close #2152

### CODE REVIEW

File: packages/strapi-plugin-graphql/hooks/graphql/index.js [hunk 1]
```diff
@@ -54,7 +54,7 @@ module.exports = strapi => {
         // Load APIs configurations.
         new Promise((resolve, reject) => {
           glob(
-            './api/*/config/*.graphql',
+            './api/*/config/*{.graphql,.graphql.js}',
```
Comment (maturanomx) [inline packages/strapi-plugin-graphql/hooks/graphql/index.js]: 🤔 Why not add just the optional extension?; it the real objective isn't?. Not tested, but I think it should be something like:

```
'./api/*/config/*.graphql?(.js)'
```

Comment (greatwitenorth) [reply packages/strapi-plugin-graphql/hooks/graphql/index.js]: Yeah that might make more sense. I'm not so good with globs 😄

**Comment by Aurelsicoko** [comment]
@greatwitenorth Can you test the solution proposed by @maturanomx? It should work, then we'll be able to merge the PR. Thank you for your work 👍

**Comment by greatwitenorth** [comment]
I've tested the updated glob by @maturanomx and it is working correctly. I've updated the pull request accordingly.

**Comment by maturanomx** [comment]
> 
> 
> I've tested the updated glob by @maturanomx and it is working correctly. I've updated the pull request accordingly.

Awesome!. Thanks :)

**Pull request merged**